### PR TITLE
[WEB-447] Move research & papers from org to com

### PR DESCRIPTION
**Pull request by rhyolight**

### CODE REVIEW

**Comment by rhyolight** [comment]
@lscheinkman please review

File: packages/numenta.com/pages/papers/continuous-online-sequence-learning-with-an-unsupervised-neural-network-model.md [hunk 1]
```diff
@@ -3,9 +3,120 @@ author: "Yuwei Cui, Subutai Ahmad & Jeff Hawkins"
...
 title: "Continuous Online Sequence Learning with an Unsupervised Neural Network Model"
+type: post
 ---
+
+| | |
+|-|-|
+| **Authors** | Y. Cui, S. Ahmad, and J. Hawkins |
+| **Journal** | Neural Computation 28, `2474–2504` (2016) |
+| **Online** | [Read or Download this Paper][1] |
+| **Published** | November 1, 2016 |
```
Comment (lscheinkman) [inline packages/numenta.com/pages/papers/continuous-online-sequence-learning-with-an-unsupervised-neural-network-model.md]: The **Published** date is different from the date shown at the top. Which one is correct?

**Comment by rhyolight** [comment]
I will be sure to split commits up by content and snaps next time.

File: packages/numenta.com/pages/papers/properties-of-sparse-distributed-representations.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 ---
-author: Subutai Ahmad & Jeff Hawkins
-brief: An earlier version of the above submission, this paper applies our mathematical model of sparse representations to practical HTM systems.
+author: "Subutai Ahmad & Jeff Hawkins"
+brief: "An earlier version of the above submission, this paper applies our mathematical model of sparse representations to practical HTM systems."
 category: neuroscience
 date: 2014/10/28
```
Comment (lscheinkman) [inline packages/numenta.com/pages/papers/properties-of-sparse-distributed-representations.md]: This date does not match the publication date on arxiv. It should be 2015/03/25

Comment (lscheinkman) [inline packages/numenta.com/pages/papers/properties-of-sparse-distributed-representations.md]: The date does not match the paper publication on arxiv. It should be 2015/03/25

**Pull request merged**